_pages/dev/css-architecture.md

```diff
@@ -2,74 +2,89 @@
+To deliver as much flexibility as possible for styling, all of the CSS rules are provided in a separate `@spartacus/styles` library. This makes the styles completely optional, configurable, and extensible. You can extend or replace the standard styles, and also implement your own style rules.
 
-Additionally, an existing UI framework such as Bootstrap can be used in Spartacus without making it a hardcoded dependency.
+Additionally, an existing UI framework, such as Bootstrap, can be used in Spartacus without making it a hard-coded dependency.
 
-### Style Library Versioning
+### Versioning
 
-Spartacus libraries support Semantic Versioning, which means that no breaking changes are allowed during a major version. Uncontrolled style changes in Spartacus would have a bad effect on customers, as unexpected styling could appear without notice.
+Spartacus libraries support semantic versioning, which means that breaking changes are not allowed during the life cycle of a major version. This is also true for the style library. A new style rule, or an adjusted rule, will influence the existing storefront experience. Customers who follow the semantic version scheme should not be surprised by the appearance of new styles during the life cycle of a stable release.
 
-To provide a stable style release for a major version of the library, the style library is shipped with the same semantic versioning scheme as the `core` and `component` libraries. Customers can rely on a major version of the style library, so that only new features and bugs can be expected in minor and patch releases.
-This means that the style rules for a major version cannot get any changes other then new (optional) styles (for example for a new component) or to patch a bug.
+At the same time, it is likely that Spartacus will evolve from one minor version to the next. To allow for gradual changes in the style layer, new or adjusted style rules are added for a specific version, but these changes are not added in the style build process unless you explicitly opt in to receive these changes. You need to set a single variable to leverage the latest breaking styles changes. Note that non-breaking changes are added regardless.
 
-In order to continuously improve the styles during a major release, new styles might be isolated in special theme. This theme could be shipped already during the major release. This allows to demonstrate and use right away, without interfering with existing users.
+The following example illustrates an additional style for version 2.2:
 
-The table below shows an example of some potential releases.
+```scss
+cx-mini-cart {
+  @include forVersion(2.1) {
+    background: blue;
+  }
+  @include forVersion(2.2) {
+    background: blue;
+  }
+}
+```
 
```

Comment:
The value for `background` in `forVersion(2.2)` appears to be the same as the value for `background` in `forVersion(2.1)`. Is that intentional? Or is there a missing "additional" style for version 2.2?

Reply:
Yeah, that was a copy / paste mistake. let's make one of them another color. you can choose a nice one ;)

Reply:
Haha, perfect 😄 
deepskyblue it is...